src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowPdpBuilder.java

```diff
@@ -285,6 +282,7 @@ protected String findincomeClassCode(ResultSet rs, PdpSourceRow pdpRow, String d
     @Override
     void insertNullsForTransactionRow(PreparedStatement insertStatement, TransactionDetailRow detailRow, int offset) throws SQLException {
         super.insertNullsForTransactionRow(insertStatement, detailRow, offset);
+        insertStatement.setString(detailRow.documentTitle.index - offset, null);

```

Comment:
Just to confirm, we want the document title to be null?

Reply:
If I remember correctly, the tax processing code does two passes for setting up the transaction detail rows. On the first pass, if the main data setup method doesn't explicitly set a value for a given field, then this `insertNullsForTransactionRow()` method needs to explicitly set those unset fields to null so that the SQL insert query will succeed. The code for the second pass should set these fields (including the doc title) to non-null values as needed.

Reply:
OK, thanks